### PR TITLE
Pass requested OAuth scopes

### PR DESCRIPTION
**Pull request by skbogner**
I think by mistake "read" was hardcoded in there. So even if you tried to request other scopes you'd only be allowed read access.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "publisher": "linear",
   "author": "Linear Orbit, Inc",
   "license": "MIT",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (tommoor) [inline package.json:8]: I think this should be changed as part of a separate commit that's tagged. eg through `npm version patch`

Comment (skbogner) [reply package.json:8]: Updated to have it in separate commit.
